t/Forecast.t

```diff
@@ -12,24 +12,24 @@ my $loc = test_location('de');
 
 ddg_spice_test(
     ['DDG::Spice::Forecast'],
-    DDG::Request->new(
-        query_raw => 'weather forecast',
-        location => $loc
-    ) => test_spice(
-        "/js/spice/forecast/" . uri_escape_utf8(${\$loc->loc_str}) . "/current",
-        call_type => 'include',
-        caller => 'DDG::Spice::Forecast',
-        is_cached => 0
-    ),
-    DDG::Request->new(
-        query_raw => 'forecast',
-        location => $loc
-    ) => test_spice(
-        "/js/spice/forecast/" . uri_escape_utf8(${\$loc->loc_str}) . "/current",
-        call_type => 'include',
-        caller => 'DDG::Spice::Forecast',
-        is_cached => 0
-    ),
+    #     DDG::Request->new(
```

Comment:
Same here, let's just remove this to simplify things. The docs and (hopefully) all other IA's should be providing good examples.